tf_encrypted/keras/models/sequential.py

```diff
@@ -101,3 +103,66 @@ def layers(self):
     if layers and isinstance(layers[0], InputLayer):
       return layers[1:]
     return layers[:]
+
+  def set_weights(self, keras_weights, sess):
+    """ Sets the weights of the model.
+    Arguments:
+      weights: A list of Numpy arrays with shapes and types
+          matching the output of model.get_weights()
+      sess: tfe session"""
+
+    # List model weights
+    model_weights = []
+    for l in self.layers:
+      model_weights += l.weights
+
+    # Define keras weights as private variables
+    keras_weights_p = [tf_encrypted.define_private_variable(w)
+                       for w in keras_weights]
+
+    # Assign new keras weights to existing weights defined by
+    # default when tfe layer instantiated
+    sess.run(tf.global_variables_initializer())
+    for i, w in enumerate(model_weights):
+      sess.run(tf_encrypted.assign(w, keras_weights_p[i]))
```

Comment:
@mortendahl  - couple question: 

- I am currently defining the keras weights  as  private variable in `set_weights` before assigning it to the tfe weights. I'd like to move this define_private_variable into `tfe.assign`, however I think I will need to have the Session in `tfe.assign` to initialize the weights `sess.run(tf.global_variables_initializer())`. Do you have any recommendation to approach this? 
- currently `set_weights` expects a tfe session. It's a bit different from the keras api. However I am not sure if  `get_tfe_session()` would work in this context.

Reply:
hmm, why would you create variables inside `tfe.assign`? it sounds like we're working around another issue here that should be fixed instead

Reply:
good point

Reply:
`with tfe.Session() as sess` directly in `set_weights` to assign new weights doesn't give the right output. I guess because it's not re-using the same session when running the model with the following code:

```
with tfe.Session() as sess:
    sess.run(tf.global_variables_initializer())
    
    tfe_model.set_weights(keras_weights)
    
    y = tfe_model(x)
    actual = sess.run(y.reveal())
```